### PR TITLE
Issue docstrings

### PR DESCRIPTION
**Pull request by PhysicsFTW**
Creating docstrings for all methods.

### CODE REVIEW

File: pocs/utils/too/email_parser/email_parser.py [hunk 1]
```diff
@@ -17,9 +17,31 @@
...
+        - imap_host (str): name of email hosting service.
+        - imap_user (str): the email address name.
+        - imap_pass (str): the email password.
+        - test_message (bool): enables the reading of test messages.
+        - checked_targets (python list): list of final targets in the parser.
+        - mail (IMAP4_SSL Object): email object accessed by given hostname,
+            email and password.
+        - alert_pocs (bool): Tells the Alerter class whether or not to send checked_targets.
+
+    TODO:
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: Turn these TODO items into actual Issues and remove from docstring. (Plus I think the first one is done)

File: pocs/utils/too/alert_pocs.py [hunk 2]
```diff
@@ -5,36 +5,54 @@
 class Alerter(object):
 
     def __init__(self, port_num=6500, *args, **kwargs):
+        '''Uses PanMessaging to send a list of targets as a list of python dictionaries.
+
+        Attributes:
+            port_num = 6500 by default (int): the publisher port number.
+            verbose = False by default (bool): tells the methods whether or not to print.
+
+        TODO:
```
Comment (wtgee) [inline pocs/utils/too/alert_pocs.py]: Turn TODO item into an Issue and remove from docstring.

File: pocs/utils/too/email_parser/email_parser.py [hunk 3]
```diff
@@ -143,11 +179,31 @@ def parse_event(self):
 class GravWaveParseEmail(ParseEmail):
 
     def __init__(self, *args, **kwargs):
+        '''EmailParser object which handles the gravity wave specific emails.
+
+        Attribs:
+            - selection_criteria (python dict): example: {'name': (srt), 'max_tiles': (float)}, determines
+                when tiling is complete. If not provided, is read from config_grav.
+            - observer (astroplan Observer Object): an observer with the location specified in the
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: Write this as `astroplan.Observer` (with the dot). Smart documentation software will then automatically link it.

File: pocs/utils/too/email_parser/email_parser.py [hunk 4]
```diff
@@ -143,11 +179,31 @@ def parse_event(self):
 class GravWaveParseEmail(ParseEmail):
 
     def __init__(self, *args, **kwargs):
+        '''EmailParser object which handles the gravity wave specific emails.
+
+        Attribs:
+            - selection_criteria (python dict): example: {'name': (srt), 'max_tiles': (float)}, determines
+                when tiling is complete. If not provided, is read from config_grav.
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: What is `config_grav`? I dont' see it anywhere.

File: pocs/utils/too/alert_pocs.py [hunk 5]
```diff
@@ -5,36 +5,54 @@
 class Alerter(object):
 
     def __init__(self, port_num=6500, *args, **kwargs):
+        '''Uses PanMessaging to send a list of targets as a list of python dictionaries.
```
Comment (wtgee) [inline pocs/utils/too/alert_pocs.py]: Whenever you refer to an actual class or method you should put it in back-ticks `` `PanMessaging` ``

File: pocs/utils/too/email_parser/email_parser.py [hunk 6]
```diff
@@ -165,8 +221,16 @@ def read_email(self, text):
         return more_split_msg
 
     def parse_event(self, text):
-        '''After read_email returns the python dictionary, this method craetes all the parameters
-        to pass to GravityWaveEvent, which then handles the target creation.'''
+        '''Interprets Gravity Wave email and creates list of targets.
+
+        After read_email returns the python dictionary, this method craetes all the parameters
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: `` `read_email` ``

File: pocs/utils/too/email_parser/email_parser.py [hunk 7]
```diff
@@ -165,8 +221,16 @@ def read_email(self, text):
...
+        to pass to GravityWaveEvent, which then handles the target creation.
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: `` `GravityWaveEvent` ``

File: pocs/utils/too/email_parser/email_parser.py [hunk 8]
```diff
@@ -237,10 +305,17 @@ def parse_event(self, text):
 class SupernovaParseEmail(ParseEmail):
 
     def __init__(self, *args, **kwargs):
+        '''Email Parser object which handles the Supernoave specific emails.'''
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: Supernoave -> Supernovae (there's one up above too)

File: pocs/utils/too/email_parser/email_parser.py [hunk 9]
```diff
@@ -271,6 +352,15 @@ def read_email(self, text):
         return more_split_msg
 
     def parse_event(self, text):
+        '''Interprets Supernova email and creates list of targets.
+
+        After read_email returns the python dictionary, this method craetes the target list.
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: `` `read_email` ``

File: pocs/utils/too/email_parser/email_parser.py [hunk 10]
```diff
@@ -330,6 +430,15 @@ def read_email(self, text):
         return more_split_msg
 
     def parse_event(self, text):
+        '''Interprets GRB email and creates list of targets.
+
+        After read_email returns the python dictionary, this method craetes the target list.
```
Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: `` `read_email` ``

File: pocs/utils/too/horizon/horizon_range.py [hunk 11]
```diff
@@ -10,9 +10,15 @@
...
 
 
-class Horizon():
+class Horizon(object):
 
     def __init__(self, observer, altitude, time=current_time(), *args, **kwargs):
+        '''Contains methods to work out the RA and DEC coordinates for the observable range from the set observer.
+
+        Attribs:
+            - observer (astroplan Observer Object): Required in the init.
```
Comment (wtgee) [inline pocs/utils/too/horizon/horizon_range.py]: `` `astroplan.Observer` `` - also don't need the 'Object'

File: pocs/utils/too/horizon/horizon_range.py [hunk 12]
```diff
@@ -10,9 +10,15 @@
...
+            - altitude (float): the altitude (in degrees) below which we do not want to observe.
+            - time (astropy.time.Time Object): time at which we want to observe.'''
```
Comment (wtgee) [inline pocs/utils/too/horizon/horizon_range.py]: Don't need 'Object'

File: pocs/utils/too/email_parser/email_parser.py (hunk 4 above)
Comment (PhysicsFTW) [inline pocs/utils/too/email_parser/email_parser.py]: It's not an attribute, it's the name of the email_parsers config we want to read. Basically configname.

Comment (wtgee) [inline pocs/utils/too/email_parser/email_parser.py]: I mean you refer to `config_grav` twice but that is not referenced anywhere else and I have no idea what it is.

**Pull request merged**